assets/js/modules/ads/datastore/base.js

```diff
@@ -29,13 +29,25 @@ import {
 } from './settings';
 
 const baseModuleStore = Modules.createModuleStore( 'ads', {
-	ownedSettingsSlugs: [ 'conversionID', 'paxConversionID', 'extCustomerID' ],
+	ownedSettingsSlugs: [
+		'conversionID',
+		'paxConversionID',
+		'customerID',
+		'extCustomerID',
+		'formattedExtCustomerID',
+		'userId',
+		'accountOverviewURL',
+	],
```

Comment:
These owned settings should match ones that we have in the settings class.

Reply:
Have corrected, thank you